manager2/src/app/my-delete-confirm/my-delete-confirm.component.ts

```diff
@@ -26,8 +27,11 @@ export class MyDeleteConfirmComponent implements OnInit {
     cancel() {
         this.isDeleting = false;
     }
```

Comment:
Should reset message on cancel

Reply:
74ff07309d7856f7f3a3b9dd26608423af18193c